### DISCUSSION TITLE
Docker - marking nodes offline  - use of closed network connection

### DISCUSSION THREAD

**Discussion by PeS82** [minio/minio#19837] [Q&A] [answered]
Hi,

I'm testing minio using docker compose, pretty much [the official sample one](https://github.com/minio/minio/blob/master/docs/orchestration/docker-compose/docker-compose.yaml).

When uploading larger files from the console UI, I'm getting these errors in the minio docker logs:

> Error: Marking minio1:9000 offline temporarily; caused by Post "http://minio1:9000/minio/storage/data1/v57/createfile?disk-id=e26fd058-1a1c-44f1-a682-0285c04de36d&file-path=1a711507-f88a-4bc6-851a-79433b13071f%2F2f5e6ec2-35cd-4f7f-bbe7-455ef70b380a%2Fpart.1&length=201741070&orig-volume=tetsik&volume=.minio.sys%2Ftmp": write tcp 172.18.0.4:49070->172.18.0.3:9000: use of closed network connection (*fmt.wrapError)

The upload shows `503` and then `500` errors in the nginx log, the upload fails with

> Error: Resource requested is unwritable, please reduce your request rate

Minio is behind [nginx proxy](https://min.io/docs/minio/linux/integrations/setup-nginx-proxy-with-minio.html) that is however hosted on another machine with SSL terminated in the nginx. Underlying system is Ubuntu. The version is: RELEASE.2024-05-28T17-19-04Z.

What am I missing? Any settings I should adjust?

Thanks in advance.

**Comment by harshavardhana** [comment]
This looks like a bunch of network issues on your end.

**Reply by PeS82** [reply]
Within the docker network? It shows IMHO it fails when communicating between the minio containers. The containers are on the same machine, next to each other, communicating via the network that is created by docker compose locally.

I don't want to argue, just purely confused here. Thanks.

**Reply by harshavardhana** [reply, marked as answer]
Yeah mostly because of some underlying problems, a sample is not meant for uploading large files. It creates 8 virtual drives on a single drive locally it will eat up I/O and can cause load on the drives.

Depending on the system it will behave differently based on how much your setup can tolerate. 

If you want to test large objects bring some real setup with real drives.

**Reply by PeS82** [reply]
Thank you. One last question - is it ok to use docker at all for "production" once we get there?

**Reply by harshavardhana** [reply]
yes definitely @PeS82
